webapp/src/app.tsx

```diff
@@ -2951,24 +2980,25 @@ export class ProjectView
         }
         files[pxt.CONFIG_NAME] = pxt.Package.stringifyConfig(cfg);
 
-        await pxt.github.cacheProjectDependenciesAsync(cfg);
-
-        const hd = await workspace.installAsync({
-            name: cfg.name,
-            meta: {},
-            editor: options.preferredEditor || options.prj.id,
-            pubId: "",
-            pubCurrent: false,
-            target: pxt.appTarget.id,
-            targetVersion: pxt.appTarget.versions.target,
-            cloudUserId: this.getUserProfile()?.id,
-            temporary: options.temporary,
-            tutorial: options.tutorial,
-            extensionUnderTest: options.extensionUnderTest,
-            isSkillmapProject: options.skillmapProject
-        }, files);
-
-        await this.loadHeaderAsync(hd, { filters: options.filters });
+        const hd = await workspace.installAsync(
+            {
+                name: cfg.name,
+                meta: {},
+                editor: options.preferredEditor || options.prj.id,
+                pubId: "",
+                pubCurrent: false,
+                target: pxt.appTarget.id,
+                targetVersion: pxt.appTarget.versions.target,
+                cloudUserId: this.getUserProfile()?.id,
+                temporary: options.temporary,
+                tutorial: options.tutorial,
+                extensionUnderTest: options.extensionUnderTest,
+                isSkillmapProject: options.skillmapProject
+            },
+            files
+        );
+
+        await this.loadHeaderAsync(hd, { filters: options.filters }, false);
```

Comment:
Just curious... does disabling the cloud sync check fix something?

Reply:
oh right, i forgot to list that in the changes! this is a new project, so no need to do a down sync here (it doesn't exist yet)